festim/stepsize.py

```diff
@@ -0,0 +1,28 @@
+import festim as F
+
+
+class Stepsize:
+    """
+    A class for evaluating the stepsize of transient simulations.
+
+    Args:
+        initial_value (float, int): initial stepsize.
+
+    Attributes:
+        initial_value (float, int): initial stepsize.
+    """
+
+    def __init__(
+        self,
+        initial_value,
+    ) -> None:
+        self.initial_value = initial_value
+
+    def get_dt(self, mesh):
+        """Defines the dt value
+        Args:
+            mesh (dolfinx.mesh.Mesh): the domain mesh
+        Returns:
+            fem.Constant: the dt value
+        """
+        return F.as_fenics_constant(self.initial_value, mesh)
```

Comment:
I don't know how this will play out when we want to start adapting it...

Wondering if it the dt (as a `fem.Constant)` should live in `Stepsize` or in `HTransportProblem`

Let's keep it like this but be aware that we might have to change things a bit

Reply:
notes